benchmark/merlin_hashtable_benchmark.cc.cu

```diff
@@ -177,7 +177,7 @@ float test_one_api(const API_Select api, const size_t dim,
   options.dim = dim;
   options.max_hbm_for_vectors = nv::merlin::GB(hbm4values);
   options.io_by_cpu = io_by_cpu;
-  options.evict_strategy = EvictStrategy::kCustomized;
+  options.evict_strategy = EvictStrategy::kLru;
```

Comment:
I have a question whether use kLru or KCustimized  as the benchmark is depend on what.

Reply:
The `Lru` would be more popular according to recent feedback from end-users.